### ISSUE TITLE
Provide More Feedback on Paragraph Formatting

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4494]
**Reported by netblue on 2014-09-26 22:26**
Hi, it is great that NVDA can now report paragraph formatting but it would be very useful if NVDA, could report the following:

Indent a paragraph from the left CTRL+M

Remove a paragraph indent from the left CTRL+SHIFT+M

Create a hanging indent CTRL+T

Reduce a hanging indent CTRL+SHIFT+T

Remove paragraph formatting CTRL+Q

As you can see, these commands are used to quickly apply formatting and user efficiency would improve significantly if NVDA could have this capability.
If NVDA already supports this and for some reason I missed it. I would like to inform that it does not work in Spanish version of Word.
Note: for development reasons I only mentioned the English  commands, of Corse if this feature was to be added then translation is necessary.

**Comment by ehollig** [comment]
This works correctly for me using Microsoft Office Word 2016 (English). Testing performed:

1. Open MS Word
2. Type a sentence, "The brown fox jumps over the lazy dog."
3. Perform one of the above commands, (Example: CTRL+M, CTRL+Shift+M, CTRL+T, CTRL+Shift+T, CTRL+Q)
4. After performing one of the above commands, report formatting with NVDA+F
### Expected and Actual Behavior
NVDA correctly announces the indentation of the paragraph with the command performed.

The original poster, however, is using Word in a different language. I am unsure if this would make a difference. CC @michaelDCurran

**Comment by zstanecic** [comment]
Confirmed:
It doesn’t work
Nvda version: 2017.3 rc1, polish Microsoft office 2016
Using windows in polish, too
Steps to reproduce used as said
Od: Ethan Holliger
Wysłano: sobota, 19 sierpnia 2017 02:43
Do: nvaccess/nvda
DW: Subscribed
Temat: Re: [nvaccess/nvda] Provide More Feedback on Paragraph Formatting(#4494)

This works correctly for me using Microsoft Office Word 2016 (English). Testing performed:
1. Open MS Word
2. Type a sentence, "The brown fox jumps over the lazy dog."
3. Perform one of the above commands, (Example: CTRL+M, CTRL+Shift+M, CTRL+T, CTRL+Shift+T, CTRL+Q)
4. After performing one of the above commands, report formatting with NVDA+F
Expected and Actual Behavior
NVDA correctly announces the indentation of the paragraph with the command performed.
The original poster, however, is using Word in a different language. I am unsure if this would make a difference. CC @michaelDCurran

**Comment by Adriani90** [comment]
It is properly reported when pressing NVDA+f. I am using Word 2016 in german. However, I think the user wanted it to be reported when pressing the key stroke. Not when pressing nvda+f.

**Comment by Adriani90** [comment]
cc: @paulber007

**Comment by CyrilleB79** [comment]
Seems a duplicate of #6269

**Comment by Adriani90** [comment]
Thanks for picking this @CyrilleB79. I am closing this one as duplicate.
